hazelcast-sql/pom.xml

```diff
@@ -71,6 +71,73 @@
                 <version>${maven.shade.plugin.version}</version>
                 <executions>
                     <execution>
+                        <id>fat-jar</id>
```

Comment:
As far as I understand, these changes are needed for a convenient development setup for Jet. Could you please explain, how exactly the development setup will look like?
Another question - does this change introduce any changes to IMDG SQL artifacts produced during a default build?

Reply:
There will be 2 artifacts produced:
- hazelcast-sql-{version}.jar, shaded dependencies etc. (no changes here)
- hazelcast-sql-{version}-jar-with-depepndencies.jar, dependencies included but not shaded, it will be used by Jet, so we get the same version of Calcite and other libraries as IMDG while being able to work with classes from Calcite (non shaded) packages